README.md

```diff
@@ -2,6 +2,34 @@
 
 ## Running locally
 
+Before running the application locally in the form of a Docker container, any already running instances of the application (also in the form of a Docker container) should be stopped to avoid port conflicts, unless
+other ports have been assigned to each application instance. 
+
+To run the application in the form of a Docker container, run the command `docker compose up -d` from the root of the application directory.
+
+The database engine containing the `timecard schema` should be running already.
+
+#### Flow
+
+First, Docker Compose will run the container containing Liquibase, which will perform any necessary DB migrations. This container will be killed automatically. The container logs contain a message about the success/failure of the migration execution.
+
+Then the timecard-restapi application in the form of a container will be launched, but it will be possible to edit the application code. Changes will be visible almost automatically thanks to hot-reload.
+
+**Restarting the container** after making changes to the application or building its new image is not necessary or advisable.
+
+
+### Devtools Hot Deployment in local environment 
+
+Devtools allows you to reload the application after changing the state of the file, but to see the effect of the changes you need to do one of the following options:
+
+ - 1 - Build project manually (InteliJ IDEA: Build/Build Project )
+ - 2 - [or] IntelliJ IDEA has 2 properties that will allow you to execute `Build Project` automatically. 
+   1) Go to `Preferences/Build,Execution,Deployment/Compiler` and select option
+      `Build project automatically`
+   2) [Optional] Go to `Preferences/Advanced Settings` and select `Allow auto-make to start even if developed application is currently running`
+
+
+
 ### Authenticate with the Home Office private Maven repository (Artifactory)
 
 In order to retrieve private Maven packages, you’ll need to [configure authentication for Artifactory](https://collaboration.homeoffice.gov.uk/display/EAHW/Artifactory).
```

Comment:
We'll also need to cover the issue that this is problematic on minikube as it doesn't support the VPN out of the box

Reply:
Added info re: running in on minikube